src/main/scala/cromwell/engine/backend/local/LocalEngineFunctions.scala

```diff
@@ -15,7 +15,9 @@ class LocalEngineFunctions(executionContext: TaskExecutionContext) extends Engin
   override protected def read_int(params: Seq[Try[WdlValue]]): Try[WdlInteger] = {
     assertSingleArgument(params).map {
       case f: WdlFile =>
-        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.toInt)
+        WdlInteger(io.Source.fromFile(f.value.toFile).mkString.trim.toInt)
```

Comment:
Aha, I think I figured this out. I think that's the function which gets called when you call .close on the object. Trace it through a bit, it gets tacked on with withClose to the BufferedSource, it doesn't appear to have any smarts on autoclosing.

Either that or the accumulated knowledge of the ancients is incorrect, although that certainly happens sometimes